app/controllers/courses_controller.rb

```diff
@@ -121,7 +121,7 @@ def save_deadlines
   end
 
   def enable
-    authorize! :teach, @organization
+    authorize! :teach, @organization # should assistants be able to enable/disable?
```

Comment:
As I notice @jamox already said above..

I agree, OK to leave for now but should definitely refactor if we ever want to develop permissions further.